[DOCS] Exceptions UX updates (#2808)
elastic/security-docs: docs/detections/shared-exception-lists.asciidoc

Apply shared exception lists to rules:

. Go to *Manage* -> *Shared Exception Lists*.
. Do one of the following:
** Select a shared exception list and click *Manage rules*. 
** Find the shared exception list you want to assign to rules, click the *More actions* menu (*...*), then select *Manage rules*.

> . Choose rules to add the exception to. Click the toggle in the Link column to select a rule.

Author: Thanks for these suggestion, @joepeeples! The step for opening the details page of a shared exception list definitely needed to be more precise.

[[manage-exception-lists]]
=== Manage shared exception lists

You can edit, export, import, and delete shared exception lists from the Shared Exception Lists page.  

> NOTE: After upgrading to 8.6, exception lists created in 8.5 and earlier are imported as shared exception lists. You can access imported exception lists from the Shared Exception Lists page.  

Reviewer: I would probably opt for `saved as` vs. `imported as` here if we truly are not importing.

> NOTE: After upgrading to 8.6, exception lists created in 8.5 and earlier are imported as shared exception lists. You can access imported exception lists from the Shared Exception Lists page.  

Author: Thanks @joepeeples and @jmikell821 for the thoughtful feedback! 

@joepeeples you're correct in that rules and exceptions don't need to be imported as part of the upgrade process when moving from 7.x -> 8.x or to a newer version of 8.x. Now, there is [an upgrade scenario](https://www.elastic.co/guide/en/security/8.6/upgrade-intro.html#reenable-rules-upgrade) where users need to re-enable disabled rules, but this case is entirely separate from the situation described in this note.

Going back to how "old" exception lists (i.e. exception lists created in 8.5 or earlier) are treated in 8.6+: when users upgrade to 8.6+, their "old" exception lists are kept within the system and no backend actions (e.g., migrating, importing, or saving) are applied to them. They're just kept in the same place and then displayed within the new Shared Exceptions List page after the upgrade. 

Since there are no backend changes to "old" exception lists, maybe it'd make sense to instead describe how exception lists are perceived or handled on the frontend after users upgrade to 8.6+. For example:

```suggestion
NOTE: Exception lists created in 8.5 and earlier are treated as shared exception lists in 8.6 or later. You can access all shared exception lists from the Shared Exception Lists page. 
```

Let me know your thoughts if you'd recommend additional changes. TY!

> NOTE: After upgrading to 8.6, exception lists created in 8.5 and earlier are imported as shared exception lists. You can access imported exception lists from the Shared Exception Lists page.  

Author: Another, more literal, revision:

```suggestion
NOTE: Exception lists created in 8.5 and earlier are displayed as shared exception lists in 8.6 or later. You can access all shared exception lists from the Shared Exception Lists page.   
```

> NOTE: After upgrading to 8.6, exception lists created in 8.5 and earlier are imported as shared exception lists. You can access imported exception lists from the Shared Exception Lists page.  

Reviewer 2: This edit looks OK, except I wonder if "are treated as" could just be something more definitive like "become." 

Saying that the old lists "are treated as" shared lists sounds like the old lists still exist and might behave in their old way somehow, but they're just "treated as" new in this context. But AFAIK from the user's perspective, it's simpler: the old lists are now the new shared lists?

> NOTE: After upgrading to 8.6, exception lists created in 8.5 and earlier are imported as shared exception lists. You can access imported exception lists from the Shared Exception Lists page.  

Reviewer 2: Ah, I see you edited while I was commenting. I think the same applies to "displayed as" -- it still sounds like we're saying the lists exist one way, but _appear_ another way. I don't know we need to convey that kind of distinction or nuance.

> NOTE: After upgrading to 8.6, exception lists created in 8.5 and earlier are imported as shared exception lists. You can access imported exception lists from the Shared Exception Lists page.  

Author: ```suggestion
NOTE: Exception lists created in 8.5 and earlier become shared exception lists in 8.6 or later. You can access all shared exception lists from the Shared Exception Lists page.    
```
